docs/acme-integration.rst

```diff
@@ -156,28 +156,44 @@ configuration active, it will check for validity of the certificate, and
 about a month before the expiration date it will try to renew the certificate
 automatically.
 
-Certificate for subdomains excluding the apex domain
-----------------------------------------------------
+Example: Certificate for apex domain and subdomains
```

Comment:
Hm, certificates can also be issued for a CNAME/PTR in DNS so I don’t think that *host* is better. But I understand your point. @drybjed What do you think?